docs/pages/concepts/offloading/mutating-webhook.md

```diff
@@ -0,0 +1,91 @@
+---
+title: Mutating webhook
+weight: 5
+---
+
+This section explains how the Liqo Webhook mutates Pods created within a Liqo-enabled namespace, enabling them to be possibly offloaded to remote clusters and enforcing user-specified constraints.
+
+As detailed in the [namespace offloading](/usage/namespace_offloading) section, the creation of a *NamespaceOffloading* resource inside a given namespace extends that namespace to remote clusters (possibly remapped, to prevent naming conflicts).
+Additionally, each *NamespaceOffloading* resource states the offloading constraints specified by the user, and leveraged by the webhook to appropriately mutate the Pods in that namespace (i.e., adding the *toleration* to the virtual nodes *taint*, and modifying the *node affinities* field).
+In particular:
+
+1. The [PodOffloadingStrategy](/usage/namespace_offloading#selecting-the-pod-offloading-strategy) field, which determines at a high-level whether Pod offloading is enabled.
+2. The [ClusterSelector](/usage/namespace_offloading#selecting-the-remote-clusters) field, which optionally selects a subset of virtual clusters for Pod offloading.
+
+### Pod mutations
+
+This section shows in detail the webhook behavior with each of the three possible `PodOffloadingStrategy` values, while considering a fixed `ClusterSelector` that targets all virtual nodes with label `liqo.io/region=us-west-1`:
+
+```yaml
+clusterSelector:
+  nodeSelectorTerms:
+  - matchExpressions:
+    - key: liqo.io/region
+      operator: In
+      values:
+      - us-west-1
+```
+
+#### LocalAndRemote PodOffloadingStrategy
+
+The `LocalAndRemote` strategy allows Pods to be scheduled both on local nodes and a subnet of the virtual nodes.
+Hence, the webhook adds the *toleration* to the virtual nodes *taint*, as well as mutates the pod affinity introducing two (ORed) `NodeSelectorTerms`:
```

Comment:
I felt hard to understand what ORed was.
I would suggest to replace the parenthesis with "(in logical OR)".